packages/common/src/styles/RTLProvider.tsx

```diff
@@ -4,5 +4,16 @@ import { useRtl } from '../intl/intlHelpers';
 export const RTLProvider: FC = props => {
   const isRtl = useRtl();
 
-  return <div style={{ direction: isRtl ? 'rtl' : 'ltr' }} {...props} />;
+  return (
+    <div
+      style={{
+        height: '100vh',
+        width: '100vw',
+        display: 'flex',
+        flexDirection: 'column',
+        direction: isRtl ? 'rtl' : 'ltr',
+      }}
+      {...props}
+    />
+  );
```

Comment:
I think this was silly. I forgot that chucking a `div` in here would mess things up and it would be easier to add the `direction` straight onto the `body`